PCPTool.v11/dll/PCPWbcl.cpp

```diff
@@ -696,7 +696,7 @@ Return value:
             // Move to the first log entry after the descriptor.
             // WbclApiMoveToNextElement() does boundary checks.
```

Comment:
Does the reference in this comment need to change as well?  
It could be confusing to leave a call that is similar but not the same.